src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -274,6 +274,8 @@ public class DeploymentOptions
 
         private Dictionary<string, string> _displayNameMapDict;
 
+        public Dictionary<string, int> IncludeObjectsTable;
```

Comment:
it's a little confusing that this has Table in the name but it's a Dictionary. Maybe just call it IncludeObjects?